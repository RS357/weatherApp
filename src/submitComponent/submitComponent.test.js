import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json'
import SubmitComponent from './submitComponent.js';



describe('SubmitComponent should', () => {
    it ('exist', () => {
        expect(SubmitComponent).toBeDefined()   
    });

    it('have a consistent structure', () => {
        const wrapper = mount(<SubmitComponent />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})