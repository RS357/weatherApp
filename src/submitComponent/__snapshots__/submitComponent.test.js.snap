// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitComponent should have a consistent structure 1`] = `
<SubmitComponent>
  <input
    type="submit"
    value="submit"
  />
</SubmitComponent>
`;
